sdk/eventhub/event-hubs/test/client.spec.ts

```diff
@@ -70,15 +70,21 @@ function arrayOfIncreasingNumbersFromZero(length: any): Array<string> {
 
 before("validate environment", function(): void {
   should.exist(
-    process.env.EVENTHUB_CONNECTION_STRING,
+    env[EnvVarKeys.EVENTHUB_CONNECTION_STRING],
     "define EVENTHUB_CONNECTION_STRING in your environment before running integration tests."
   );
-  should.exist(process.env.EVENTHUB_NAME, "define EVENTHUB_NAME in your environment before running integration tests.");
+  should.exist(
+    env[EnvVarKeys.EVENTHUB_NAME],
+    "define EVENTHUB_NAME in your environment before running integration tests."
+  );
 });
 
-const service = { connectionString: process.env.EVENTHUB_CONNECTION_STRING, path: process.env.EVENTHUB_NAME };
+const service = {
+  connectionString: env[EnvVarKeys.EVENTHUB_CONNECTION_STRING],
+  path: env[EnvVarKeys.EVENTHUB_NAME]
+};
 
-describe("EventHubClient on ", function(): void {
+describe(`EventHubClient on ${runnableInBrowserMarker}`, function(): void {
```

Comment:
When any new test is added, the person needs to opt in to whether they want to run the test in the browser and add the add the corresponding tag. Having the tag as a constant and then using template literals feels like an overkill. Can we include the hashtag directly here? It will also help with readability.

Reply:
By overkill, is there any specific disadvantage that may be there?

The disadvantage I see and why would be in favor of centralizing this string is:
- to avoid textual errors 
- We may choose to add more tags and managing them manually would take up more development time.
- be able to find them more easily using the reference

Reply:
For readability, do you mean of the code or test-case messages at runtime? For code, we can improve the name of variable if the current one seems hard to follow.

Reply:
I agree on the advantages listed above if and when we have we have the extra tags to deal with. Right now, we only need to identify tests that are to be run in both node and browser vs node only tests. Are there other scenarios?

What is the use case for the `notYetRunnableInBrowserMarker`? Doesnt the absence of `runnableInBrowserMarker` ensure that the tests will not run in the browser? If it is to track the tests that should run in browser, but don't at the moment, then we can track them with issues. Why have a dedicated marker for it in the code?

For this PR, I would prefer to be in sync with what the browser tests in Service Bus are doing. If and when we decide to change the hash tag or use constants, then that can be done in a separate PR targeting both Event Hubs and Service Bus. This way, at any snapshot in time i.e at any commit, we are following the same procedure of running tests in the 2 projects.

> For readability, do you mean of the code or test-case messages at runtime

By readability, I meant the code and not the output at runtime.

Reply:
Since we want to run all the tests in browser, it would be easier to mark Node only tests. Shall we have hashtags for just these instead?

Reply:
Let's make that change in a separate PR as a combined effort across Event Hubs and Service Bus

Reply:
Or create a new issue

Reply:
Updated PRand logged #3474